usr/src/cmd/zfs/zfs_main.c

```diff
@@ -4469,6 +4473,9 @@ parse_fs_perm(fs_perm_t *fsperm, nvlist_t *nvl)
 			avl_pool = fspset->fsps_who_perm_avl_pool;
 			avl = fsperm->fsp_uge_avl;
 			break;
+
+		default:
+			break;
```

Comment:
i have added assertion if avl_pool = NULL.
we have this situation only with perm_type = ZFS_DELEG_WHO_UNKNOWN